File: Java/392.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        if(s.length() == 0) return true;
        int k = 0;
        int n = t.length();
        for(int i=0;i<n;i++){
            if(s.charAt(k) == t.charAt(i)) k++;
            if(k == s.length()) return true;
        }
        return false;
    }
}

File: Python/392.py
class Solution(object):
    def isSubsequence(self, s, t):
        if len(s) == 0:
            return True
        k = 0
        for i in t:
            if s[k] == i:
                k += 1
            if k == len(s):
                return True
        return False